lib/index_group.rb

```diff
@@ -21,6 +21,13 @@ def initialize(base_uri, name, schema, search_config)
       @search_config = search_config
     end
 
+    def find_index(real_name)
+      # TODO: validate this against the API to make sure it actually exists
```

Comment:
Oops, I forgot about this. I was thinking of issuing a basic get request, since the API will return 404 if the index doesn't exist. It will do that anyway if you try do actually do anything with this index, but thought the behaviour was a bit surprising.

Will take another look.